overlord/assertstate/bulk.go

```diff
@@ -175,7 +175,7 @@ func bulkRefreshValidationSetAsserts(s *state.State, vsets map[string]*Validatio
 		}
 	}
 
-	err := resolvePoolNoFallback(s, pool, beforeCommitChecker, userID, deviceCtx)
+	err := resolvePoolNoFallback(s, pool, beforeCommitChecker, userID, deviceCtx, true)
```

Comment:
Using a variable would make it more clear what the last parameter is for:
```suggestion
	autoRefresh := true
	err := resolvePoolNoFallback(s, pool, beforeCommitChecker, userID, deviceCtx, autoRefresh)
```